ksqldb-rest-app/src/main/java/io/confluent/ksql/rest/server/execution/ConnectExecutor.java

```diff
@@ -81,6 +94,45 @@ public static StatementExecutorResponse execute(
         .map(err -> new ErrorEntity(statement.getStatementText(), err)));
   }
 
+  private static List<String> validate(final CreateConnector createConnector,
+      final ConnectClient client) {
+    final Map<String, String> config = new HashMap<>(createConnector.getConfig().size());
+    createConnector.getConfig().forEach((k, v) ->
+        // Parsing the statement wraps string fields with single quotes which breaks the
```

Comment:
The unit tests added in the first version of this PR didn't catch this bug (that we weren't stripped quotes from configs before passing them for validation), right? Is there test coverage somewhere in the codebase that prevents a regression here? Maybe ConnectIntegrationTest.java?

Reply:
Yes, this bug was caught by `ConnectIntegrationTest`. `ConnectExecutorTest` didn't catch it because we mock the connector there.